ui/_tests_/utils/testNameinCamelCase_spec.test.js

```diff
@@ -0,0 +1,8 @@
+import { CamelCaseToSentanceCase } from "../../utils/camelCaseToSentanceCase";
+
+describe('CamelCase String Utility', () => {
+  test('make camelCase to sentansecase', () => {
+    const val = CamelCaseToSentanceCase("mesheryComponents");
+    expect(val).toBe("meshery  Components")
+  })
```

Comment:
Are you sure we will have double space between meshery and Components? 
I see tests are passing so curious to know why we are having double space, is it intentional or nitpik? 🤔